### DISCUSSION TITLE
Native Instruments FM8 for 9.91€ , Pluginboutique.com

### DISCUSSION THREAD

**Discussion by Banana71** [probonopd/MiniDexed#532] [General]
FM inspiration for little money, for everyone who loves FM synthesis. Until 08/31/23

https://www.pluginboutique.com/product/1-Instruments/4-Synth/8001-FM8?currency=EUR

**Comment by probonopd** [comment]
Thanks for the hint. That's quite a saving, however I'm happy with Dexed on the PC. Do you know, can it do unison (like the Korg [opsix native](https://www.korg.com/products/software/opsix_native/))?

**Comment by Banana71** [comment]
> Do you know, can it do unison (like the Korg [opsix native](https://www.korg.com/products/software/opsix_native/))?

Yes, Unison is present. Up to 64 voices, pan and detune are adjustable.

**Reply by probonopd** [reply]
OK, that's a killer feature for me :+1: Thanks for checking.

**Comment by GregWinfield** [comment]
damn. thats 17 bucks AUD instead of $219. i think thats worthwhile!!!!!!!!!!
